src/components/FileUpload/index.tsx

```diff
@@ -10,16 +10,18 @@ const FileUpload = (props: TFileUploadProps): JSX.Element | null => {
     allowedTypes = ['*'],
     label,
     getFiles,
+    removeFiles,
     name,
     setFieldValue,
     toBase64,
     required,
     disabled,
     isFileUploaded,
```

Comment:
this shouldn't be prop

Reply:
why if you do something in the parent component  when you remove the file